src/veritysetup/veritysetup.c

```diff
@@ -78,7 +82,18 @@ static int parse_options(const char *options) {
                 else if (streq(word, "panic-on-corruption"))
                         arg_activate_flags |= CRYPT_ACTIVATE_PANIC_ON_CORRUPTION;
 #endif
-                else
+                else if ((val = startswith(word, "root-hash-signature="))) {
+
+                        r = free_and_strdup(&arg_root_hash_signature, val);
+                        if (r < 0)
+                                return log_oom();
+                } else if (stat(word, &st) == 0 || (val = startswith(word, "base64:"))) {
```

Comment:
nit: let's invert this check, so that we don't try and stat weird non-existing paths if base64:foo is passed

Reply:
Good nit indeed.